Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                var name = GetString("What is your name: ");
+                var surname = GetString("What is your surname: ");
+                var age = GetInt("What is your age: ");
+
+
+                var weight = GetFloat("What is your weight in kilogram: ");
+                var height = GetFloat("What is your height in metres: ");
+                var bmi = GetBmi(weight, height);
+
+                Console.Write($"{name}  {surname}  is {age} years old, his weight is {weight} kg and his height is {height:F2} cm.");
+                Console.WriteLine($"His BMI is:  {bmi:N2}");
+                Console.WriteLine("");
+                if (i == 0)
+                {
+                    Console.WriteLine($"Now for another person {Environment.NewLine}  ");
+                }
+            }
+        }
+
+        public static float GetBmi(float weight, float height)
+        {
+            if (height <= 0 || weight <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+
+                if (height <= 0 && weight <= 0)
+                {
+                    Console.WriteLine($"Weight cannot be equal or less than zero, but was 0.{Environment.NewLine}Height cannot be less than zero, but was 0.");
+                }
+                else if (height <= 0)
+                {
+                    Console.WriteLine($"Height cannot be equal or less than zero, but was {height}.");
+                }
+                else if (weight <= 0)
+                {
+                    Console.WriteLine($"Weight cannot be equal or less than zero, but was {weight}.");
+                }
```

Comment:
This works.  If you wanted it to be slightly more compact you could do:

                if (weight <= 0)
                {
                    Console.WriteLine($"Weight cannot be equal or less than zero, but was {weight}.");
                }
                if (height <= 0)
                {
                    Console.WriteLine($"Height cannot be equal or less than zero, but was {height}.");
                }

Using `if` instead of `else if` it will print both messages if both statements are true so you don't need the combined one.

Reply:
I disagree. The combined case has a different output. There are ways to solve this in a similar fashion though.

Reply:
then im curious how to make the nested if then 's prettier. If when it was not booleans but for example numbers or strings I could have used switch.

Reply:
I agree with @ellmaster.

Reply:
I also because what ellsmaster says the output on both cases are not the same

Reply:
Take a look at https://repl.it/repls/AnxiousPurpleGraduate

I get the exact same output with both styles.

Reply:
sorry n but you did not look well to the output: 

Console.WriteLine($"Weight cannot be equal or less than zero, but was {weight}.{Environment.NewLine}Height cannot be equal or less than zero, but was {height}.");

is not equal to : 

Console.WriteLine($"Weight cannot be equal or less than zero, but was {weight}.{Environment.NewLine}Height cannot be equal or less than zero, but was {height}.");

as ellmaster already said. the second part is different.